SimPEG/Optimization.py

```diff
@@ -939,6 +943,72 @@ def approxHinv(self, value):
         self._approxHinv = value
 
     @Utils.timeIt
+    def minimize(self, evalFunction, x0):
```

Comment:
cool, I think this looks good! is there a test that runs it? 
@rowanc1 : what do you think?